docs/doc_sources/api_reference/dpctl/sycl_usm_array_interface.rst

```diff
@@ -32,23 +32,23 @@ Python dictionary with the following fields:
     is read-only).
 
 ``"strides"``
-    an optional tuple of integers describing number of array elements needed to jump
+    An optional tuple of integers describing number of array elements needed to jump
     to the next array element in the corresponding dimensions. The default value of ``None``
     implies a C-style contiguous (row-major compact) layout of the array.
 
 ``"offset"``
-    an optional Python integer encoding offset in number of elements from the pointer
+    An optional Python integer encoding offset in number of elements from the pointer
     provided in ``"data"`` field to the array element with zero indices. Default: `0`.
 
 ``"syclobj"``
     Python object from which SYCL context to which represented USM allocation is bound.
 
     ==============================================  =======================================
-    Filter selector string                          Platform's default context for platform
+    Filter selector string                          Platform's default context
                                                     the SYCL device selected by the
                                                     :ref:`filter selector string <filter_selector_string>`
-                                                    is a part of.
-    :py:class:`dpctl.SyclContext`                   An explicitly provided context
+                                                    .
```

Comment:
This edit change the meaning of the description. In case "syclobj" keyword argument is a filter selector string, the USM is bound to the default platform context of the platform to which the device selected by the filter selector string belongs.